src/main/java/com/edmunds/tools/databricks/maven/BaseDatabricksMojo.java

```diff
@@ -197,11 +214,92 @@ String createDeployedArtifactPath() throws MojoExecutionException {
         String modifiedDatabricksRepo = databricksRepo;
         String modifiedDatabricksRepoKey = databricksRepoKey;
         if (databricksRepo.endsWith("/")) {
-            modifiedDatabricksRepo = databricksRepo.substring(0, databricksRepo.length()-1);
+            modifiedDatabricksRepo = databricksRepo.substring(0, databricksRepo.length() - 1);
         }
         if (databricksRepoKey.startsWith("/")) {
             modifiedDatabricksRepoKey = databricksRepoKey.substring(1, databricksRepoKey.length());
         }
         return String.format("%s%s/%s", DEFAULT_DBFS_ROOT_FORMAT, modifiedDatabricksRepo, modifiedDatabricksRepoKey);
     }
+
+    public String stripPrefix(String path) {
```

Comment:
hmm it does seem like it would be best to leave all of this in a separate class. What is the rationale for moving this all into base mojo?

Reply:
thanks for run down. I think I honed in on the 1st option further down. What are the negatives of that approach?

Reply:
Originally, it looked like the only way would be to duplicate the logic that actually does the stripping. Now that I've taken another look, I think I found a better way. I'm going to close this and take another pass. Thanks!